tests/test_merge.py

```diff
@@ -0,0 +1,138 @@
+import pytest
```

Comment:
See earlier comments. We should have several test files at this point:
- **test_merge.py**: Runs tests using the `confluence merge ...` CLI interface and direct calls to `confluence.subcommands.merge.main`.
- **test_io.py**: Verifies Excel, JSON, CSV, and text IO operations.
- **test_validate.py**: Runs tests using the `confluence validate ...` CLI interface and direct calls to `confluence.subcommands.validate.main`.
- **test_validate_callbacks.py**: Tests `confluence.core.validator` callback functions.